recipients_app/lib/view/widgets/income/balance_card/balance_card_bottom_action.dart

```diff
@@ -0,0 +1,123 @@
+import "package:app/data/models/payment/balance_card_status.dart";
+import "package:app/data/models/payment/payments_ui_state.dart";
+import "package:app/ui/buttons/button_small.dart";
+import "package:app/ui/configs/configs.dart";
+import "package:flutter/material.dart";
+
+class BalanceCardBottomAction extends StatelessWidget {
+  final PaymentsUiState paymentsUiState;
+
+  const BalanceCardBottomAction({
+    super.key,
+    required this.paymentsUiState,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final foregroundColor = _getForegroundColor(paymentsUiState.status);
+
+    return Container(
+      color: _getBackgroundColor(paymentsUiState.status),
+      child: Padding(
+        padding: AppSpacings.a16,
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Expanded(
+              child: Text(
+                _getStatusLabel(paymentsUiState),
+                style: TextStyle(color: foregroundColor),
+              ),
+            ),
+            Row(
+              children: [
+                ButtonSmall(
+                  onPressed: () {},
+                  label: _getPrimaryActionLabel(paymentsUiState),
+                  buttonType: ButtonSmallType.outlined,
+                  color: foregroundColor,
+                  fontColor: foregroundColor,
+                ),
+                if (shouldShowSecondaryActionButton(
+                  paymentsUiState,
+                )) ...[
+                  const SizedBox(width: 8),
+                  ButtonSmall(
+                    onPressed: () {},
```

Comment:
This should send something to firebase in the future, right? Maybe add a comment